### PR TITLE
Second pull request

### PR DESCRIPTION
**Pull request by deikdesign**

### CODE REVIEW

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

  ### Hi team, you guys did a very good job so far and Your page meets most of the requirements  👏 
  #### I can see that you guys are working hard at this project, and you're almost done, there are just some required changes* that you guys have to implement in order to proceed:

  _Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.
  
## [REQUIRED CHANGES]
- [ ] Remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) 
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
image

- [ ] Guys, next time take care with the way you write some commits, they should be descriptive so the other programmer can see exactly what you changed with that commit. [Check this link](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
image
(What you did change here? added something? removed something? Please be more descriptive next time 💪)


## [OPTIONAL CHANGES]
- [ ] Replace Author1 and Author2 with your names (To make your repo more professional)

##Summary
✨Your project is looking really nice, congrats 💯
✨ Good job using semantic HTML! I really enjoyed that 👏
🔧 You guys just have to make the requested changes listed above! Should be really easy to you both 💪

## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀

  👨🏻‍💻 **TSE: Kevin Alves**

    

  📫 Have any question? Better call Kevin:

  <a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

    
  <sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

File: style.css [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  width: 25px;
+  height: 25px;
+}
+
+.header-text {
+  margin-left: 5px;
+  margin-right: 10px;
+}
+
+/********************************   */
```
Comment (SafaErden) [inline style.css]: - Please remove redundant comments.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                    and have read and acknowledge our
+                    <a href="https://www.intuit.com/privacy/">Privacy Statement</a>. </p>
+            </div>
+        </div>
+        <div id="container-recaptcha">
+            <p class="recaptcha text-center"> Invisible reCAPTCHA by Google
+                <a href="https://www.mint.com/terms">Privacy Policy</a> and
+                <a href="https://policies.google.com/terms?hl=en">Terms of Use</a></p>
+        </div>
+    </section>
```
Comment (SafaErden) [inline index.html:82]: - [OPTIONAL] Please consider the use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

**Review by SafaErden — COMMENTED** [review]
## Status: Require changes ♻️ 🚥 

Hi **TEAM**, your project is awesome 👍 it really looks exactly like the original. 👐 However, we need a little fix before we can proceed. Kindly find comments below:

<hr>

- [ ] You have a failed linter check. Please fix it.
image

- [ ] Please consider updating the background image position inside the password input file.
image


<hr>

As I mentioned above, your project is almost done. You have some minor issues to solve. Please be an optimist and do not forget that the main purpose is to learn and practice. Do not let the requested changes discourage you 👍

<hr>

Kindly make these changes and submit another code review.

Happy coding_ 👨‍💻 

If you have any question, you can contact me on Slack :point_right: @safaerden

File: README.md [hunk 3]
```diff
@@ -1 +1,39 @@
...
+[Live Demo Link](https://deikdesign.github.io/form-project-mint/)
+
+Prerequisites - Chrome browser
+*Setup
+ - $ cd "folder location"
+ - $ git clone https://github.com/deikdesign/form-project-mint.git
+* Install
+* Usage 
+* Run tests - HTML validator, w3 validation
+* Deployment
```
Comment (SpaYco) [inline README.md]: - [x]  [Optional] please either update this section with the needed information or delete it

**Review by SpaYco — APPROVED** [review]
## Project Approved ✔️ 

good job! your project is approved, however I have some suggestions I listed down below 😉 


- [ ] try making your image sources shorter, Hint: save the images locally or in an image host

- [ ] as the previous TSE suggested, Please consider the use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

- **I listed more down below**

### don't forget to merge! good luck with the next projects 😄 

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**